src/reference/es/00-Getting-Started/01-Setup/c.md

```diff
@@ -7,42 +7,96 @@ out: Installing-sbt-on-Linux.html
   [RPM]: $sbt_rpm_package_base$sbt-$app_version$.rpm
   [DEB]: $sbt_deb_package_base$sbt-$app_version$.deb
   [Manual-Installation]: Manual-Installation.html
-
-<!-- TODO: Translate to Spanish -->
+  [website127]: https://github.com/sbt/website/issues/127
+  [cert-bug]: https://bugs.launchpad.net/ubuntu/+source/ca-certificates-java/+bug/1739631
 
 Installing sbt on Linux
 -----------------------
 
-### Installing from a universal package
+### Instalar desde un paquete universal
 
-Download [ZIP][ZIP] or [TGZ][TGZ] package and expand it.
+Descarga el paquete [ZIP][ZIP] o [TGZ][TGZ] y descomprímelo.
 
-### RPM and DEB
+### Instalar JDK
 
-The following packages are also officially supported:
+Primero desberás de instalar JDK. Recomendamos Oracle JDK 8 u OpenJDK 8.
+Los detalles sobre el nombre de los paquetes cambian de una distribución a otra.
 
-  - [RPM][RPM] package
-  - [DEB][DEB] package
+Por ejemplo, Ubuntu xenial (16.04LTS) usa
+[openjdk-8-jdk](https://packages.ubuntu.com/hu/xenial/openjdk-8-jdk).
 
-> **Note:** Por favor reporte cualquier problema que se tenga con los paquetes
-> arriba mencionados al projecto
-> [sbt-launcher-package](https://github.com/sbt/sbt-launcher-package).
+La familia Redhat lo llama
+[java-1.8.0-openjdk-devel](https://apps.fedoraproject.org/packages/java-1.8.0-openjdk-devel).
 
-### Gentoo
+### Ubuntu y otras distribuciones basadas en Debian
+
+Los paquetes [DEB][DEB] son oficialmente soportados por sbt.
+
+Ubuntu y otras distribuciones basadas en Debian usan el formato DEB, pero por lo
+general no necesitas instalar software desde un fichero DEB local.
+En su lugar lo que se utiliza son los gestores de paquetes, tanto desde la línea de
+comandos (p.e. `apt-get`, `aptitude`) o con una interfaz gráfica de usuario
+(p.e. Synaptic).
+Ejecuta lo siguiente desde el terminal para instalar `sbt`
+(necesitarás tener privilegios de administrador para hacerlo, de ahí el `sudo`).
+
+    echo "deb https://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list
+    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2EE0EA64E40A89B84B2DF73499E82A75642AC823
+    sudo apt-get update
+    sudo apt-get install sbt
+
+Los gestores de paquetes utilizan los repositorios para buscar los paquetes que
+se desean instalar. Los binarios de sbt son publicados en Bintray y
+convenientemente Bintray proporciona un repositorio APT.
+Sólo tienes que añadir el repositorio en aquellos ficheros utilizados por tu
+gestor de paquetes.
+
+Una vez `sbt` haya sido instalado podrás gestionar el paquete en `aptitude` o
+Synaptic después de que hayas actualizado la caché de paquetes. También podrás
+ver el repositorio recién añadido al final de la lista en
+Preferencias del sistema -> Software y actualizaciones -> Otro software:
+
+![Ubuntu Software & Updates Screenshot](files/ubuntu-sources.png "Ubuntu Software & Updates Screenshot")
```

Comment:
I think this needs to change to `(../files/ubuntu-sources.png "Ubuntu Software & Updates Screenshot")` because `es/` directory is parallel with `files/`.

That might explain the pandoc errors:
```
[error] pandoc: Could not find image `files/ubuntu-sources.png', skipping...
[error] pandoc: Could not find image `files/setting-expression.png', skipping...
[error] pandoc: Could not find image `files/rgb_color_solid_cube.png', skipping...
[error] pandoc: Could not find image `files/sbt-configurations.png', skipping...
[error] pandoc: Could not find image `files/task-dependency00.png', skipping...
[error] pandoc: Could not find image `files/task-dependency01.png', skipping...
[error] pandoc: Could not find image `files/task-dependency02.png', skipping...
[error] pandoc: Error producing PDF from TeX source.
[error] ! Unable to load picture or PDF file 'files/ubuntu-sources.png'.
[error] <to be read again> 
[error]                    }
[error] l.266 \includegraphics{files/ubuntu-sources.png}
```

Reply:
You're right.
Also there are more broken links (like those pointing to the API ref) I'll also fix them.